Transbank/WebpayRest/TransaccionCompleta/TransaccionCompleta.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using Transbank.Common;
+using Transbank.Webpay.Common;
+
+namespace Transbank.Webpay.TransaccionCompleta
+{
+    public class TransaccionCompleta

```

Comment:
It seems as this class should be merged in the FullTransaction class. I can not see reasons to have these configurations in a separate class.

Reply:
fixed at 71f3587